src/main/kotlin/com/github/hd/tornadofxsuite/view/Dialog.kt

```diff
@@ -29,8 +29,8 @@ class Dialog : Fragment() {
                     vboxConstraints { marginRight = 5.0 }
 
                     listview<String> {
-                        if (scanner.mapClassViewNodes.size > 0) {
-                            scanner.mapClassViewNodes.forEach { (className, digraph) ->
+                        if (fxTestGenerator.scanner.mapClassViewNodes.size > 0) {
```

Comment:
I don't think you need to check the size. I don't think `forEach` will run unless the size is greater than 0.

Reply:
lol Jonathan pointed out a similar thing for me previously. I have to double check with how that would work with HashMaps but I will likely apply your suggestion